### DISCUSSION TITLE
[Exemplars] Optimize exemplars tooltip visualization

### DISCUSSION THREAD

**Discussion by aishyandapalli** [grafana/grafana#72406] [Observability]
When there are too many labels for an exemplar, hovering on an exemplar tooltip visualization is overflown. Also, the **traceid**  label is added at the end. So, we need to scroll the tooltip till below and then click on the link to jump to traces. This visualization is getting a little tricky to jump to traces.

Can we add a configuration section for exemplars where we can configure the order of exemplar labels and also hide few exemplar labels which we don't need?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82628. Feel free to continue the discussion around this there. Thank you!
